identifiers/country-fr/districts.csv

```diff
@@ -0,0 +1,578 @@
+id,name
+ocd-division/country:fr/oversea_collectivity:mf/ed:1,1ère circonscription de St-Barthélemy/St-Martin
+ocd-division/country:fr/oversea_collectivity:nc/ed:1,1ère circonscriptio de Nouvelle-Calédonie

```

Comment:
Looks like a typo on this line and the one below: circonscriptio -> circonscription

Reply:
Done.